### DISCUSSION TITLE
[Question] Deploy method other than VPS?

### DISCUSSION THREAD

**Discussion by oknchm** [anasty17/mirror-leech-telegram-bot#1227] [Q&A]
Is there a way to deploy the leech bot without a VPS? For example, on Windows or maybe Google Colab?

Thank you!

**Comment by junedkh** [comment]
Use docker for desktop

**Comment by anasty17** [comment]
There is a docker support. So when u can use docker then u can deploy this repo

**Comment by tania010** [comment]
this might be a stupid question but, whose network bandwidth is used with "docker for desktop" e.g. while downloading files or uploading them to drive, will the speed and bandwidth be limited to what my internet connection can provide? or will it be provided by docker (backend(?) if there is such a thing) similar to what heroku did?
